src/Posts/Post.jsx

```diff
@@ -180,6 +195,29 @@ const CommentWrapper = styled.div`
   }
 `;
 
+const TextLink = styled("Link")`
+  display: in-line block;
+  margin: 0;
+  font-size: 14px;
+  line-height: 18px;
+  color: ${(p) => (p.bold ? "#11181C !important" : "#687076 !important")};
+  font-weight: ${(p) => (p.bold ? "600" : "400")};
+  font-size: ${(p) => (p.small ? "12px" : "14px")};
+  overflow: ${(p) => (p.ellipsis ? "hidden" : "visible")};
+  text-overflow: ${(p) => (p.ellipsis ? "ellipsis" : "unset")};
```

Comment:
As Caleb suggested some time ago, let's use styled-components properties with `$` to not to add this properties to the DOM:

```suggestion
  color: ${(p) => (p.$bold ? "#11181C !important" : "#687076 !important")};
  font-weight: ${(p) => (p.$bold ? "600" : "400")};
  font-size: ${(p) => (p.$small ? "12px" : "14px")};
  overflow: ${(p) => (p.$ellipsis ? "hidden" : "visible")};
  text-overflow: ${(p) => (p.$ellipsis ? "ellipsis" : "unset")};
```